pkg/translator/loki/logs_to_loki.go

```diff
@@ -131,6 +132,19 @@ func LogsToLokiRequests(ld plog.Logs) map[string]PushRequest {
 	return requests
 }
 
+func getFormatFromFormatHint(logAttr pcommon.Map, resourceAttr pcommon.Map) string {
+	format := formatJson
+	formatVal, found := resourceAttr.Get(hintFormat)
+	if !found {
+		formatVal, found = logAttr.Get(hintFormat)
+	} else {
```

Comment:
What's up with the empty else statement?

Reply:
removed!